config/sync/core.extension.yml

```diff
@@ -39,13 +39,16 @@ module:
   filter: 0
   flag: 0
   geolocation: 0
-  handy_cache_tags: 0
```

Comment:
I think we want to keep this module enabled: it's handy to provide more granular cache tags for views listings

Reply:
Good catch, recently added modules. This will be one thing to watch as the pace picks up.